src/main/java/lotto/view/ResultView.java

```diff
@@ -1,24 +1,23 @@
 package lotto.view;
 
-import lotto.domain.Lotto;
 import lotto.domain.LottoGame;
 import lotto.domain.LottoResult;
 import lotto.domain.Rank;
 
-import java.util.List;
-
 public class ResultView {
-    public static void printLottos(List<Lotto> lottos) {
-        System.out.println(lottos.size() + "개를 구매했습니다.");
-        for (Lotto lotto : lottos) {
-            System.out.println(lotto);
-        }
+    public static void printLottos(LottoGame lottoGame) {
+        System.out.println(lottoGame.lottoCount() + "개를 구매했습니다.");
+        System.out.println(lottoGame);
     }
 
     public static void printResult(LottoResult result, LottoGame lottoGame) {
```

Comment:
```suggestion
    public static void printResult(LottoResult result, Money money) {
```
로직을 실행하는데 반드시 필요한 값만 전달하는 것은 어떨까?